woocommerce/woocommerce-ios
# [Payments Menu] New payments menu feature flag

Part of: #11103


## Description


This lays the groundwork for the rewrite of the Payments menu in Swift UI.

In order to easily test the differences between the two screens, we want to be able to toggle between them while the app is running. This adds a feature flag and Experimental Feature toggle for the new screen, as well as an empty implementation.

## Testing instructions


1. Run the app from Xcode
2. Navigate to `Menu > Payments`
3. Observe that the normal payments menu is shown
4. Navigate to `Menu > Settings > Experimental Features` and toggle the new payments menu
5. Navigate to `Menu > Payments`
6. Observe that an empty `Hello World` screen is shown.


Delete the app, then test that the new screen isn't shown with a release build, and can't be toggled on.

---
